src/ucp/core/ucp_worker.c

```diff
@@ -958,13 +958,11 @@ static void
 ucp_worker_select_best_ifaces(ucp_worker_h worker, ucp_tl_bitmap_t *tl_bitmap_p)
 {
     ucp_context_h context = worker->context;
-    ucp_tl_bitmap_t tl_bitmap;
+    ucp_tl_bitmap_t tl_bitmap = UCS_BITMAP_INIT;
```

Comment:
align =
(pls check clang result)

Reply:
Aligned manually. It wasn't in clang output.